pkgs/tools/admin/awscli2/default.nix

```diff
@@ -51,7 +51,6 @@ with py.pkgs; buildPythonApplication rec {
 
   propagatedBuildInputs = [
     bcdoc
-    botocore
```

Comment:
You sure this is intended?

Reply:
Yeah. I'm fairly certain this is not needed because there isn't a botocore >= 2 package yet:
https://search.nixos.org/packages?channel=unstable&from=0&size=30&sort=relevance&query=botocore

Reply:
I've reverted this line but still confused why it works in both cases.

Reply:
If it is really not needed it should work in both cases.

Reply:
It _is_ a required package, so it should be left in IMO.
https://github.com/aws/aws-cli/blob/f7da416ba174be85c488eca90e56237b6d3305a0/setup.py#L27